### PR TITLE
Only select the URL bar on navigation if the page is the homepage, and

### LINKED ISSUE #1521: Don't select the URL bar when loading a page
* Load a page.
* Click a link.

Expected:
* The page itself has focus, so Space scrolls, Backspace navigates back, etc.

Actual:
* The URL bar has focus. Hitting any key will modify the visible URL.

### PR DESCRIPTION
**Pull request by jsantell**
do not select the URL bar on history forking. Fixes #1521. r=vporof

I think focusing on did-navigate and did-stop-navigation is for selecting it upon a new tab. As a follow up, we should probably mark page models whether or not they have navigated at all to determine focus (if I manually type in tofino://history, I would expect it to not get focused), but that's a bit more of an edge case.

### CODE REVIEW

**Comment by jsantell** [comment]
Adding stubs since forking page sagas aren't tested at all

**Comment by jsantell** [comment]
The fork tests will be easier once #1543 lands too with the url utility

File: test/unit/browser-modern/sagas/page-sagas-test.js [hunk 1]
```diff
@@ -177,4 +177,7 @@ describe('page sagas', () => {
 
     expect(gen.next().done).toEqual(true);
   });
+
+  it('should be able to fork a page back in history');
+  it('should be able to fork a page forward in history');
```
Comment (victorporof) [inline test/unit/browser-modern/sagas/page-sagas-test.js]: Why just stubs? It should be fairly straightforward to write proper tests here.

**Comment by jsantell** [comment]
@victorporof added saga test

File: test/unit/browser-modern/sagas/page-sagas-test.js [hunk 2]
```diff
@@ -177,4 +178,36 @@ describe('page sagas', () => {
...
+        location: forkedLocation,
+      }), { id: forkedId }))
+    );
+
+    expect(gen.next().value).toEqual(
+      put(PageActions.setPageIndex(forkedId, 6))
+    );
+    expect(gen.next().value).toEqual(
+      put(PageActions.setSelectedPage(forkedId))
+    );
```
Comment (victorporof) [inline test/unit/browser-modern/sagas/page-sagas-test.js]: Make sure the generator ends here via `expect(gen.next().done).toEqual(true);`

File: test/unit/browser-modern/sagas/page-sagas-test.js [hunk 3]
```diff
@@ -177,4 +178,36 @@ describe('page sagas', () => {
...
+    gen.next(5); // getPageIndexById
+
+    const createPageSessionCall = gen.next(history).value;
+    const forkedId = createPageSessionCall.CALL.args[0].id;
+    const forkedLocation = URLUtil.createHistoryRestoreUrl(history, 0);
+
+    expect(createPageSessionCall).toEqual(
+      call(PageSagas.createPageSession, Object.assign(PageEffects.createPageSession({
+        location: forkedLocation,
+      }), { id: forkedId }))
```
Comment (victorporof) [inline test/unit/browser-modern/sagas/page-sagas-test.js]: You can pass in an id to `createPageSession`, why use Object.assign?

Comment (jsantell) [inline test/unit/browser-modern/sagas/page-sagas-test.js]: +1
